docs/blog/2019-01-30-why-themes/index.md

```diff
@@ -0,0 +1,82 @@
+
+The weakness of our abstractions to date, however, is that plugins are all at the same level of abstraction meaning plugins can’t compose other plugins. Which means there’s no way to ship a group of plugins and configuration designed to solve more complex requirements e.g. a blog which needs markdown processing, pagination, RSS feed, manifest, etc.
+
+The level of a given abstraction should be matched to the granularity of a developer’s intent. Many developers _do_ want fine-grained control over how they build their sites. They’re happy to dig into how Gatsby’s data layer works, learn GraphQL, play around with plugins, etc. Especially if Gatsby is a tool they’ll be using extensively to build many types of sites. But for many people, they just want a blog and are frustrated that Gatsby makes it difficult to get that.
+
+Plugins have been a very successful lower-level abstraction for Gatsby. Themes is our new higher-level abstraction on the [ladder of website abstractions](http://worrydream.com/LadderOfAbstraction/) that will enable a new wave of innovation and usage as people use the abstraction to develop website "products" based on Gatsby they can open source or sell.
+
+## Who’s it for?
+
+- People new to Gatsby that want to get started quickly without a difficult learning curve
+
+- People who have clear and common use-cases in mind for Gatsby (e.g. blog, ecommerce, documentation) and want to share the complexity of maintaining the underlying functionality with others — so pick a theme as the base for their site instead of starting from scratch
+
+- OSS/Commercial entrepreneurs who want to create web products. Themes will enable them to assemble a product they can share or sell with others.
+
+- Designers who want to create unique/beautiful website designs to share with others
+
+## Why build it?
+
+We want everyone in the world using Gatsby.
+
+To do this, we need to make building sites with Gatsby a lot easier so they’re accessible to far more people and use cases.
+
+Many users really love Gatsby because it lets them accomplish complex tasks with ease.
+
+But for many other users, that want easy solutions to a common problems, they find Gatsby too difficult.
+
+Some tweets to that effect:
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">I want to make a really basic docs website. Ideally I just want to write<br> markdown and CSS. Nothing fancy, everything static. Trying out <a href="https://twitter.com/gatsbyjs?ref_src=twsrc%5Etfw">@gatsbyjs</a> but feel intimidated with the usage of GraphQL in the boilerplate project and having to use JS to write everything up.</p>&mdash; Sebastian McKenzie (@sebmck) <a href="https://twitter.com/sebmck/status/1085279417151057920?ref_src=twsrc%5Etfw">January 15, 2019</a></blockquote>
+
+---
```

Comment:
Are these `hr`s for contextual grouping?

Reply:
yeah